spec/support/preload_values.rb

```diff
@@ -0,0 +1,33 @@
+# Example usage:
+#   expect { MyModel.do_the_queries }.to preload_values(:attribute, value)
```

Comment:
Also, `value` in this definition should probably be `value[s]` as it seems based on the implementation that it can either be a single value, or array of values.

Reply:
Hope docs did all this.
the definition did change a little as your question felt a little like a "by the way, did you also check..."